src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -85,6 +85,20 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
             }
           }
         } ~
+        pathPrefix("tags") {
+          pathEnd {
+            requireUserInfo() { _ =>
```

Comment:
nit: if you don't need the output of `requireUserInfo()`, you can just omit the `_ =>`.